src/app/content/hooks/locationChange/resolveContent.ts

```diff
@@ -31,6 +31,29 @@ export default async(
   return {book, page};
 };
 
+export function getContentPageReferences(htmlContent: string) {
```

Comment:
i didn't mean to move this whole function, just the `defaultVersion` part that you added to it